include/teqp/models/model_potentials/2center_ljf.hpp

```diff
@@ -0,0 +1,278 @@
+#pragma once
+
+#include "teqp/types.hpp"
+#include "teqp/exceptions.hpp"
+
+namespace teqp {
+
```

Comment:
Probably there should be a namespace here that the models should live in to avoid polluting the top-level namespace